### PR TITLE
Adding support for sklearn.preprocessing.Binarizer

### PR DESCRIPTION
**Pull request by ksaur**

### CODE REVIEW

File: hummingbird/ml/operator_converters/_binarizer_implementations.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    Class implementing Binarizer operators in PyTorch.
+    """
+
+    def __init__(self, threshold, device):
+        super(Binarizer, self).__init__()
+        self.transformer = True
+        self.threshold = torch.nn.Parameter(torch.FloatTensor([threshold]), requires_grad=False)
+
+    def forward(self, x):
+        return torch.gt(x, self.threshold).float()
```
Comment (interesaaat) [inline hummingbird/ml/operator_converters/_binarizer_implementations.py]: Do we actually need cast to float?

File: hummingbird/ml/operator_converters/_binarizer_implementations.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# -------------------------------------------------------------------------
```
Comment (interesaaat) [inline hummingbird/ml/operator_converters/_binarizer_implementations.py]: Usually we put into `_*_implementations.py` all the implementations shared between onnx and pytorch. In this case we don't have an onnx implementation, so I think we can just have the implementation into the `binarizer.py` file.

Comment (ksaur) [reply hummingbird/ml/operator_converters/_binarizer_implementations.py]: I considered this, but do we have an example currently that is just in a .py?  Not for SKL.  Maybe a few for onnx?

Comment (interesaaat) [reply hummingbird/ml/operator_converters/_binarizer_implementations.py]: Yes. Like `sv` and `iforest`.

**Pull request merged**